plugins/auth0/pkg/fetch/fetch.go

```diff
@@ -141,6 +119,43 @@ func (f *Fetcher) fetchUsers(ctx context.Context, outputWriter *js.JSONArrayWrit
 	return nil
 }
 
+// return a object map to output or a boolean to skip current user.
+func (f *Fetcher) buildOutputObjects(ctx context.Context, user *management.User) (map[string]any, error) {
```

Comment:
This is a bit of a strange name for a function. Most functions build their output. It doesn't really say much about what the function does.
Consider naming it `userToJSONMap` or just `userToMap`.